src/OSPSuite.Infrastructure/Services/LoggingBuilderExtensions.cs

```diff
@@ -30,18 +37,18 @@ public static ILoggingBuilder AddFile(this ILoggingBuilder builder, string logFi
                break;
          }
 
-         //this is a workaround to avoid having the date automatically appended to the log file name
-         //(eg log_20210921.txt, when input name is simply log.txt).
-         //Setting rollOnFileSizeLimit disables that, since the first log is log.txt, the next log_001.txt and so on
-         //since we do not expect to ever reach the log limit (in the qualificationRunner fe), this results in one file
-         //1073741824 is as far as I understand the maximum possible fileSizeLimitBytes
-         builder.AddSerilog(
-           new LoggerConfiguration()
-              .MinimumLevel.ControlledBy(new LoggingLevelSwitch { MinimumLevel = serilogLevel })
-              .WriteTo.File( logFileFullPath, fileSizeLimitBytes: 1073741824, rollOnFileSizeLimit: true, restrictedToMinimumLevel: serilogLevel, outputTemplate: "[{Timestamp:yyyy-MM-dd HH:mm:ss.fff} {SourceContext:l} {Level:u}] {Message:l} {NewLine:l} {Exception}")
-             .CreateLogger()
-         );
+         var loggerConfiguration = new LoggerConfiguration().MinimumLevel.ControlledBy(new LoggingLevelSwitch { MinimumLevel = serilogLevel });
+
+         foreach (var logFile in logFileFullPaths)
```

Comment:
Looks like you removed the comment. I would keep it as it explains why those weird values are set

Reply:
you are right, I copied it to paste it underneath and then totally forgot. corrected